### DISCUSSION TITLE
Disabling linters and Rules

### DISCUSSION THREAD

**Discussion by ghost** [super-linter/super-linter#3793] [Q&A]
I am trying to get super-linter working in an Azure DevOps CI pipeline to lint our API yaml files (OpenAPI). I have spectral working but would like to tweak the linter to our own liking to stop throwing "false positives" on rules that we do not adhere to yet. The way outlined in super-linter is to copy the appropriate linter file from the TEMPLATES directory to .github/linters, which I have done. In our case, the spectral linter for OpenAPI, .openapirc.yml, so I have the file path .github/linters/openapi.rc.yml at the root of my repo.

Here are the contents of my file so far to attempt to turn off one rule:

extends: spectral:oas
overrides:
  - files:
      - "/tmp/lint/account.yaml"
    rules:
      oas3-valid-schema-example: "off"

When this runs on our Azure Devops agent, the rule above is still flagged as an error, so apparently this file is not being recognized? I also looked at the spectral documentation and it said the rule set file to use should be .spectral.yml at the root of the repo, so I tried that as well, no luck. 

I would appreciate any advice on getting this rule set file to be recognized.

Thanks in advance.

**Comment by zkoppert** [comment]
Have you set that file as a config file in the env? See https://github.com/super-linter/super-linter/blob/main/docs/using-rules-files.md#here-is-an-example
